src/defaults/default-board.ts

```diff
@@ -0,0 +1,517 @@
+            x: 3,
+            y: 2,
+          },
+          size: {
+            width: 2,
+            height: 1,
+          },
+        },
+        sm: {
+          location: {
+            x: 1,
+            y: 3,
+          },
+          size: {
+            width: 2,
+            height: 1,
+          },
+        },
+        lg: {
+          location: {
+            x: 2,
+            y: 0,
+          },
+          size: {
+            width: 2,
+            height: 1,
+          },
+        },
+      },
+    },
+    {
+      id: '47af36c0-47c1-4e5b-bfc7-ad645ee6a990',
+      name: 'Donate',
+      url: 'https://ko-fi.com/ajnart',
+      behaviour: {
+        externalUrl: 'https://ko-fi.com/ajnart',
+        isOpeningNewTab: true,
+        tooltipDescription: 'Please consider making a donation',
+      },
+      network: {
+        enabledStatusChecker: false,
+        statusCodes: ['200'],
+      },
+      appearance: {
+        iconUrl: 'https://cdn.jsdelivr.net/gh/walkxcode/dashboard-icons@master/png/ko-fi.png',
+        appNameStatus: 'normal',
+        positionAppName: 'row-reverse',
+        lineClampAppName: 1,
+        appNameFontSize: 16,
+      },
+      integration: {
+        type: null,
+        properties: [],
+      },
+      area: {
+        type: 'wrapper',
+        properties: {
+          id: 'default',
+        },
+      },
+      shape: {
+        md: {
+          location: {
+            x: 4,
+            y: 1,
+          },
+          size: {
+            width: 1,
+            height: 1,
+          },
+        },
+        sm: {
+          location: {
+            x: 2,
+            y: 4,
+          },
+          size: {
+            width: 1,
+            height: 1,
+          },
+        },
+        lg: {
+          location: {
+            x: 6,
+            y: 0,
+          },
+          size: {
+            width: 2,
+            height: 1,
+          },
+        },
+      },
+    },
+  ],
+  widgets: [
+    {
+      id: '86b1921f-efa7-410f-92dd-79553bf3264d',
+      type: 'notebook',
+      properties: {
+        showToolbar: true,
+        content:
+          '<h2><strong>Homarr\'s</strong> notebook</h2><p>Use it as your Todo list, ideas to think about, as a "getting-started" guide for your users or even as your secret journal to confess your crushes, it stays private our your <strong>Homarr</strong> instance.</p><p>The <code>notebook</code> widget focuses on usability and is designed to be as simple as possible to bring a familiar editing experience to regular users. It is based on <a target="_blank" rel="noopener noreferrer nofollow" href="https://tiptap.dev/">Tiptap.dev</a> and supports all of its features:</p><ul><li><p>General text formatting: <strong>bold</strong>, <em>italic</em>, underline, <s>strike-through</s></p></li><li><p>Headings (h1-h6)</p></li><li><p>Sub and super scripts (&lt;sup /&gt; and &lt;sub /&gt; tags)</p></li><li><p>Ordered and bullet lists</p></li><li><p>Text align</p></li></ul>',
```

Comment:
Should we change the default text of the notebook here like mentioned on discord? little "getting started" text?